ui/src/pages/Alert/AlarmFilter/index.tsx

```diff
@@ -43,28 +43,60 @@ export default function AlarmFilter({ form, type, depend }: AlarmFilterProps) {
       }));
     }
     if (type === 'obtenant') {
-      return (list as Alert.TenantsList[]).map((cluster) => ({
-        label: <span>{cluster.clusterName}</span>,
-        title: cluster.clusterName,
-        options: cluster.tenants?.map((item) => ({
-          value: item,
-          label: item,
-        })),
-      }));
+      return (list as Alert.TenantsList[])
```

Comment:
Why declare type here instead of where the variable was initialized? Do not use so many `as`

Reply:
I know, but `list` is a union type, and its type is determined by `type`.